lib/packages/core/.bilirc.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  input: "./src/index.ts",
+  output: {
+    moduleName: "b$",
+    fileName: "burstjs[min].js",
+    format: ["iife", "iife-min"],
+    dir: "./dist"
+  },
+  env: {
+    NODE_ENV: "production"
+  },
+  plugins: {
+    ['node-resolve'] : {
+      jsnext: true,
+      preferBuiltins: true,
+      browser:true
+    },
+    commonjs: {
+      namedExports: {
+        'crypto-js': [
+          'enc',
+          'SHA256',
+          'AES',
+          'lib'
+        ],
+        '@burstjs/crypto': [
```

Comment:
this is PITA... eventual deps need to be added here. Tried to do this with named exports, but had problems resolving commonjs deps then. This solution works, even not elegant. But will try again, it must be possible to do this without specifying the exports